### PR TITLE
Code review tweaks

### PR DESCRIPTION
**Pull request by oxtoacart**
@atavism Code review feedback in form of a PR.

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -1,10 +1,10 @@
 hash: 61396f9cf05b67c1701b9df2cedebda5c550d9278e0f20507f08628e1a8089b0
-updated: 2016-11-11T16:46:53.944141082-08:00
+updated: 2016-11-17T08:17:43.293888127-05:00
```
Comment (oxtoacart) [inline glide.lock]: Updated glide.lock to fix merge conflict with master.

File: lantern.go [hunk 2]
```diff
@@ -89,9 +89,8 @@ type StartResult struct {
 }
 
 type UserConfig interface {
+	config.UserConfig
```
Comment (oxtoacart) [inline lantern.go]: Embedding config.UserConfig type to automatically pull in the whole interface.

File: lantern_test.go [hunk 3]
```diff
@@ -25,6 +25,8 @@ func (c testUserConfig) AfterStart()              {}
 func (c testUserConfig) BandwidthUpdate(int, int) {}
 func (c testUserConfig) ConfigUpdate(bool)        {}
 func (c testUserConfig) ShowSurvey(survey string) {}
+func (c testUserConfig) GetUserID() int64         { return 0 }
```
Comment (oxtoacart) [inline lantern_test.go]: Implementing interface so that test compiles (it passes too).

**Comment by myleshorton** [comment]
This LGTM @oxtoacart -- @atavism hope you don't mind me merging this in!

**Pull request merged**